src/main/kotlin/com/devfloor/untitled/article/presentation/ArticleController.kt

```diff
@@ -0,0 +1,19 @@
+package com.devfloor.untitled.article.presentation
+
+import com.devfloor.untitled.article.application.ArticleFacade
+import com.devfloor.untitled.article.application.ArticleResponse
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class ArticleController(
+    private val articleFacade: ArticleFacade
+) {
+    @GetMapping("/articles/{id}")
+    @ResponseStatus(HttpStatus.OK)
+    fun show(@PathVariable id: Long): ArticleResponse =
```

Comment:
```suggestion
    fun showById(@PathVariable id: Long): ArticleResponse =
```

 이건 어때? 단건 조회라도 id 말고 다른 조건을 쓸 수도 있으니까?

Reply:
반영완료 !

Reply:
여기 준여이는 showAllByArticleId라고 했던데 comment에서 두개 다 코멘트 조회, 아티클 조회 함수면 두개 맞춰야되는거 맞지?

Reply:
> 여기 준여이는 showAllByArticleId라고 했던데 comment에서 두개 다 코멘트 조회, 아티클 조회 함수면 두개 맞춰야되는거 맞지?

이 부분 이해를 잘 못했습니다. ! 다시 한번 설명해주실수 있나요??

Reply:
아 준영이보고 말했던건데 죄송합니다!
준영이 pr보고 댓글 단거였는데 제가 id랑 articleId랑 헷갈렸어욥 !! 무시하셔도 될꺼 같아요!